functions/wordpress/gravityForms/formatGFSubmissionData.ts

```diff
@@ -0,0 +1,104 @@
+// Returns formatted form submission values as per the gravity form schema
```

Comment:
@Danishkhurshid Still looking for more, think JS doc commenting standards. The what is it, the why you'd need to use it, and the how to use it. I know you know those, but other devs that may dig into this won't.

I too would like to know those details, didn't see the WebDevStudios guys needing something to format data, but we are doing things differently. Depending on that, we'd probably want to change the file name to be something more standard, but not sure until I get those details.